bin/configurer/add-services.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -eu
+
+dir=$(dirname ${0})
+
+# Defaults are ok and create require ccd_gateway service in idam
```

Comment:
I am not sure how much value that comment adds. I'd remove it.

Reply:
Done.